app/views/layouts/application.html.erb

```diff
@@ -14,6 +14,9 @@
   <% end %>
 </head>
 <body>
+  <div class="slimmer-inside-header">
+      <h1 class="header-title">Service Manual</h1>
```

Comment:
We're trying to follow the BEM CSS pattern in this repo. `header-title` might be the block name here, with an element name of `name`. So the more BEM like CSS class could be `header-title__name`. Or `service-manual-header__title`? It's kind a case of what _feels_ right here..

Another convention we're following is for the filename to follow the block name. So for a block called 'header-title' the filename would be `_header-title.scss`.

Perhaps the the `#global-header` override stuff could be broken into it's own file which wouldn't follow the BEM pattern because it relates to markup outside of this app.